public/collaborator.doc.html

```diff
@@ -284,7 +284,7 @@ <h2 id="require-by" class="graphdoc-section__title slds-text-heading--medium sld
             </a>
             Require by
         </h2>
-        <ul class="require-by"><li><a href="createcollaborationpayload.doc.html" title="CreateCollaborationPayload - Autogenerated return type of CreateCollaboration">CreateCollaborationPayload<em>Autogenerated return type of CreateCollaboration</em></a><li></ul>
+        <ul class="require-by"><li><a href="createcollaborationpayload.doc.html" title="CreateCollaborationPayload - Autogenerated return type of CreateCollaboration">CreateCollaborationPayload<em>Autogenerated return type of CreateCollaboration</em></a><li><li><a href="project.doc.html" title="Project - ">Project<em>null</em></a><li></ul>
```

Comment:
could you walk me through (on zoom maybe) how you made this documentation and how to update it/work with it